linux/ZswapStats.h

```diff
@@ -0,0 +1,19 @@
+#ifndef HEADER_ZswapStats
+#define HEADER_ZswapStats
+/*
+htop - ZswapStats.h
+(C) 2022 htop dev team
+Released under the GNU GPLv2+, see the COPYING file
+in the source distribution for its full text.
+*/
+
+typedef struct ZswapStats_ {
+   /* maximum size of the zswap pool */
+   memory_t totalZswapPool;
```

Comment:
Header should be compile-able stand-alone. Needs to include the necessary headers.

Reply:
Fixed; I've also corrected the similar issue with `ZramStats.h` (which is what I used as a reference for the change I made).